files/build_templates/pam_limits.j2

```diff
@@ -0,0 +1,16 @@
+#THIS IS AN AUTO-GENERATED FILE
+#
+# This file generate by j2 template file: files\build_templates\pam_limits.j2
+#
+# /etc/pam.d/pam-limits settings common to all services
+# This file is included from other service-specific PAM config files,
+# and should contain a list of the authentication modules that define
+# the central authentication scheme for use on the system
+# (e.g., /etc/shadow, LDAP, Kerberos, etc.). The default is to use the
+# traditional Unix authentication mechanisms.
+#
+# here are the per-package modules (the "Primary" block)
+
+{% if sonic_enable_pam_limit == 'true' %}

```

Comment:
>sonic_enable_pam_limit [](http://example.com/codeflow?start=6&length=22)

If we set the default limit to super large, can we hardcoded `sonic_enable_pam_limit=true`?

Reply:
sonic_enable_pam_limit is a environment variable, which will control if pam load the pam_limit.so plugin.
If we set limit so super large, then load this plugin still will use some memory. so maybe when we don't want have any limit, we don't load this plugin is a better choice.